kubenow/gluster-s3/templates/gluster-s3_daemonset.yaml

```diff
@@ -0,0 +1,76 @@
+---
```

Comment:
you may need to add toleration soon. Also @andersla will implement tainting such that network can be defined at will, so it looks good with the parameter as it is now.

Reply:
I think that taints are in place now... but double check with @andersla